std_msgs/Header header
float32 state_x
float32 state_y
float32 state_theta
float32 state_dx
float32 state_dy
float32 state_dtheta